src/Results/Results.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
```

Comment:
install react-json-view(`npm i react-json-view`). Then at line 2 put in` import ReactJson from 'react-json-view' `